### PR TITLE
Nested navigation drawer

### PR DESCRIPTION
**Pull request by DadImScared**

### CODE REVIEW

**Comment by DadImScared** [comment]
[Nested navigation drawer](https://trello.com/c/WW1xqRi6/58-nested-navigation-drawer)

File: frontend/src/App.vue [hunk 1]
```diff
@@ -64,12 +79,8 @@ export default {
     return {
       appTitle: "Audio Seva Backend",
       sidebar: false,
+      subNavigation: ["List", "Allot", "Statistics"],
```
Comment (anantakrishna) [inline frontend/src/App.vue]: Is it possible to restructure the routes (make them nested) and get sub-routes from there rather than hard-coding them here?

Comment (DadImScared) [reply frontend/src/App.vue]: It appears possible but will not update the routes if addRoutes is used to add dynamic routes.

Comment (anantakrishna) [reply frontend/src/App.vue]: We are not going to use dynamic routes, so please refactor the code to build the nested menu items automatically from the routes. Think of the fact that not all the routes are to be present in the menu. I guess some metadata should be used for the routes.

Also research a little on whether there is a specialized solution for this just not to reinvent the wheel.

**Pull request merged**